ydb/docs/en/core/integrations/orm/linq2db.md

```diff
@@ -0,0 +1,503 @@
+To opt in to the new temporal types per column, set DbType on that column, e.g. `[Column(DbType = "Date32")]`. Both families can coexist in the same table.
+---
+
+### Schema generation from attributes
+
+Describe an entity using Linq To DB attributes; the provider will create a table and indexes.
+
+{% list tabs group=lang %}
+
+- C#
+
+  ```csharp
+  using LinqToDB.Mapping;
+
+  [Table(Name = "Groups", IsColumnAttributeRequired = false)]
+  [Index("GroupName", Name = "group_name_index")]
+  public class Group
+  {
+      [PrimaryKey, Column("GroupId")]
+      public int Id { get; set; }
+
+      [Column("GroupName")]
+      public string? Name { get; set; }
+  }
+  ```
+
+{% endlist %}
+
+**Generated DDL ({{ ydb-short-name }})**:
+
+```yql
+CREATE TABLE Groups (
+    GroupId Int32 NOT NULL,
+    GroupName Utf8,
+    PRIMARY KEY (GroupId)
+);
+
+ALTER TABLE Groups
+  ADD INDEX group_name_index GLOBAL
+       ON (GroupName);
+```
+
+### Schema evolution (adding a column)
+
+Add the `Department` column to the `Group` entity:
+
+{% list tabs group=lang %}
+
+- C#
+
+  ```csharp
+  [Column] public string? Department { get; set; }
+  ```
+
+{% endlist %}
+
+**Schema change (DDL):**
+
+```yql
+ALTER TABLE Groups
+   ADD COLUMN Department Utf8;
+```
+
+> Linq To DB does not manage migrations — use Liquibase/Flyway/your tool of choice. Methods like `CreateTable<T>()`/`DropTable<T>()` are convenient for tests and local development.
+
+### Relationships between entities
+
+Describe navigations using `[Association]` attributes. Example of one-to-many between `Group` and `Student`:
+
+{% list tabs group=lang %}
+
+- C#
+
+  ```csharp
+  [Table("Students")]
+  public class Student
+  {
+      [PrimaryKey, Column("StudentId")]
+      public int Id { get; set; }
+
+      [Column("StudentName")]
+      public string Name { get; set; } = null!;
+
+      [Column]
+      public int GroupId { get; set; }
+
+      // many-to-one
+      [Association(ThisKey = nameof(GroupId), OtherKey = nameof(Group.Id), CanBeNull = false)]
+      public Group Group { get; set; } = null!;
+  }
+
+  [Table("Groups")]
+  public class Group
+  {
+      [PrimaryKey, Column("GroupId")]
+      public int Id { get; set; }
+
+      [Column("GroupName")]
+      public string? Name { get; set; }
+
+      // one-to-many
+      [Association(ThisKey = nameof(Id), OtherKey = nameof(Student.GroupId))]
+      public IEnumerable<Student> Students { get; set; } = null!;
+  }
+  ```
+
+{% endlist %}
+
+**Created tables**:
+
+```yql
+CREATE TABLE Groups (
+    GroupId Int32 NOT NULL,
+    GroupName Utf8,
+    PRIMARY KEY (GroupId)
+);
+
+CREATE TABLE Students (
+    StudentId Int32 NOT NULL,
+    StudentName Utf8,
+    GroupId Int32,
+    PRIMARY KEY (StudentId)
+);
+```
+
+> Foreign keys as database-level constraints are not created by the provider. Implement referential logic at the application level or add checks with separate scripts.
```

Comment:
Why is this a comment at the bottom? This whole example doesn't seem to be doing anything.